official/utils/testing/mock.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Mock objects and related functions for testing."""
```

Comment:
The file name is has potential naming conflict with python mock lib, how about rename this file into logger_test_lib.py and put it together with logger.py?

Reply:
Good point. Changed to mock_lib, but I think it makes more sense to keep it with the testing utils, since we have a whole dir dedicated to testing, and I'd rather future mocks go in here, rather in many different files.